createFilename.m

```diff
@@ -92,47 +92,38 @@
 [~, ~, ~] = mkdir(expParameters.outputDir);
 [~, ~, ~] = mkdir(fullfile(expParameters.outputDir, modality));
 
-if isfield(cfg,'eyeTracker')
+if cfg.eyeTracker
     [~, ~, ~] = mkdir(fullfile(expParameters.outputDir, 'eyetracker'));
 end
 
 
 %% create filenames
 
 switch modality
-    
+
     case 'beh'
-        
+
         expParameters.fileName.events = ...
             [expParameters.fileName.base, runSuffix, '_events_date-' expParameters.date '.tsv'];
-        
```

Comment:
still your PR does not include Remi's last changes. I think something is missing there, you may need to pull from cpp-bids?

Reply:
I am up to date on git kraken, which 'last changes?'